mongodb-atlas/cmd/main.go

```diff
@@ -0,0 +1,253 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+
+	"github.com/hashicorp/go-plugin"
+	"github.com/icholy/digest"
+	commonconfig "github.com/opencost/opencost-plugins/common/config"
+	atlasconfig "github.com/opencost/opencost-plugins/mongodb-atlas/config"
+	atlasplugin "github.com/opencost/opencost-plugins/mongodb-atlas/plugin"
+	"github.com/opencost/opencost/core/pkg/log"
+	"github.com/opencost/opencost/core/pkg/model/pb"
+	"github.com/opencost/opencost/core/pkg/opencost"
+	ocplugin "github.com/opencost/opencost/core/pkg/plugin"
+	"golang.org/x/time/rate"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+// handshakeConfigs are used to just do a basic handshake between
+// a plugin and host. If the handshake fails, a user friendly error is shown.
+// This prevents users from executing bad plugins or executing a plugin
+// directory. It is a UX feature, not a security feature.
+var handshakeConfig = plugin.HandshakeConfig{
+	ProtocolVersion:  1,
+	MagicCookieKey:   "PLUGIN_NAME",
+	MagicCookieValue: "mongodb-atlas",
+}
+
+const costExplorerFmt = "https://cloud.mongodb.com/api/atlas/v2/orgs/%s/billing/costExplorer/usage"
+const costExplorerQueryFmt = "https://cloud.mongodb.com/api/atlas/v2/orgs/%s/billing/costExplorer/usage/%s"
+
+func main() {
+	fmt.Println("Initializing Mongo plugin")
```

Comment:
changed to be a debug log in https://github.com/sajit/opencost-plugins/pull/4, the test harness exposed a bug when the plugin prints to stdout